utils/check-ruleset-path.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+if [ -z "$RULESET_FOLDER" ]; then
+  RULESET_FOLDER="src/chrome/content/rules"
+fi
+
+# Go to git repo root; note that $GIT_DIR is .git in this case.
+if [ -n "$GIT_DIR" ]; then
+  # GIT_DIR is set, so we're running as a hook.
+  cd "$GIT_DIR" && cd ..
+elif [ ! `which git > /dev/null` ]; then
+  # Git command exists? Cool, let's CD to the right place.
+  git rev-parse && cd `git rev-parse --show-toplevel`
+else
+  # Git is required.
+  echo >&2 "ERROR: Failed to change directory. Aborted."
+fi 
+
+# Fetch the current GitHub version of HTTPS-E to compare to its master
+if [ git ls-remote upstream-for-travis ]; then
+  git remote add upstream-for-travis https://github.com/EFForg/https-everywhere.git
+  trap 'git remote remove upstream-for-travis' EXIT
+fi
+
+if [ "$TRAVIS" ]; then
+  git fetch --depth=50 upstream-for-travis master
+else
+  git fetch upstream-for-travis master
+fi
+
+COMMON_BASE_COMMIT=`git merge-base upstream-for-travis/master HEAD`
+CHANGED_FILES=`git diff --name-only $COMMON_BASE_COMMIT | grep -v $RULESET_FOLDER`
+
+EXIT_CODE=0
+
+if [ "$CHANGED_FILES" != "" ]; then
+  while read FILE; do
+    # check if changed file is actually a ruleset
+    egrep -q "^<ruleset[^>]+>" "$FILE"
```

Comment:
One additional condition we may want to add is to ensure that `$FILE` has an `.xml` filetype.  I can see the `<ruleset>` tag being used in documentation.